cmd/up/exec.go

```diff
@@ -183,7 +183,7 @@ func newEnvsGetter(hybridCtx *HybridExecCtx) (*envsGetter, error) {
 		imageEnvsGetter: &imageEnvsGetter{
 			imageGetter: registry.NewOktetoRegistry(okteto.Config{}),
 		},
-		pathGetter: os.Getenv,
+		getLocalEnvs: os.Environ,
```

Comment:
Do we really need all of them? This can cause that there might be some bugs that are hidden because local has the right env vars. I'd limit it to as few as possible

Reply:
ok, I will add the needed one to solve this issue

Reply:
3f6c57b193ef7489324e42b0a318e486ee059ae0